Stripe/STPPaymentMethodParams.m

```diff
@@ -108,4 +108,53 @@ + (nonnull NSDictionary *)propertyNamesToFormFieldNamesMapping {
              };
 }
 
+#pragma mark - STPPaymentOption
+
+- (UIImage *)image {
+    if (self.type == STPPaymentMethodTypeCard && self.card != nil) {
+        STPCardBrand brand = [STPCardValidator brandForNumber:self.card.number];
+        return [STPImageLibrary brandImageForCardBrand:brand];
+    } else {
+        return [STPImageLibrary brandImageForCardBrand:STPCardBrandUnknown];
+    }}
+
+- (UIImage *)templateImage {
+    if (self.type == STPPaymentMethodTypeCard && self.card != nil) {
+        STPCardBrand brand = [STPCardValidator brandForNumber:self.card.number];
+        return [STPImageLibrary templatedBrandImageForCardBrand:brand];
+    } else if (self.type == STPPaymentMethodTypeFPX) {
+        return [STPImageLibrary bankIcon];
+    } else {
+        return [STPImageLibrary templatedBrandImageForCardBrand:STPCardBrandUnknown];
+    }
+}
+
+- (NSString *)label {
+    switch (self.type) {
+        case STPPaymentMethodTypeCard:
+            if (self.card != nil) {
+                STPCardBrand brand = [STPCardValidator brandForNumber:self.card.number];
+                NSString *brandString = STPStringFromCardBrand(brand);
+                return [NSString stringWithFormat:@"%@ %@", brandString, self.card.last4];
```

Comment:
this should probably be localized. Okay for a follopwup

Reply:
This isn't localized in STPPaymentMethod.m or STPCard.m either, though we do localize the similar "%@ Ending In %@" strings. We should probably localize all of them.

Reply:
👍